composer.json

```diff
@@ -20,10 +20,7 @@
         "jakub-onderka/php-console-highlighter": "0.3.*"
     },
     "require-dev": {
-        "phpunit/phpunit": "~3.7|~4.0",
-        "symfony/finder": "~2.1",
-        "squizlabs/php_codesniffer": "~2.0",
-        "fabpot/php-cs-fixer": "~1.3"
```

Comment:
> By removing these from composer dev requirements, you force contributors to maintain a global installed copy that stays in sync with whatever version we expect to have. Is there any downside to leaving them here so that people can still run vendor/bin/php-cs-fixer fix before committing?

I think it's fine for people to have them globally installed otherwise we're adding time to our travis build. Some projects even expect contributors to have a global install of phpunit too.